lib/config/presets/internal/monorepo.ts

```diff
@@ -232,6 +232,7 @@ const repoGroups = {
   workbox: 'https://github.com/googlechrome/workbox',
   xterm: 'https://github.com/xtermjs/xterm.js',
   'xunit-dotnet': 'https://github.com/xunit/xunit',
+  'yarn': 'https://github.com/yarnpkg/berry',
```

Comment:
```suggestion
  yarn: 'https://github.com/yarnpkg/berry',
```

This should fix the linter problem. 😉